src/main.rs

```diff
@@ -16,7 +12,7 @@ use ziggurat_core_crawler::summary::NetworkSummary;
 use crate::{
     config::CrunchyConfiguration,
     geoip_cache::GeoIPCache,
-    ips::Ips,
+    ips::ips_algorithm::Ips,
```

Comment:
one less keyword, wdyt?
```suggestion
    ips::algorithm::Ips,
```

Reply:
Done.